example/src/qr_demo.dart

```diff
@@ -70,7 +70,7 @@ class QrDemo {
     //
     // Type Div
     //
-    for (var i = 1; i <= 10; i++) {
+    for (var i = 1; i <= 40; i++) {
```

Comment:
keep this at 10 – 40 is WAAAY too big. breaks the UI

Reply:
reverted